src/Entities/Schema.php

```diff
@@ -195,6 +202,36 @@ public function __construct()
             '3dsecure/{3dsecureId}' => function (array $content) {
                 return new ThreeDSecure($content);
             },
+            'reporting-currencies' => function (array $content) {
```

Comment:
This seems a bad conflict resolving. I believe we remove this endpoint and resource.